tests.py

```diff
@@ -32,131 +36,60 @@ def test_missing_default(self):
 
     def test_missing(self):
         "Ensures a missing file without a default raises an exception."
-        self.assertRaises(fulltext.FullTextException, fulltext.get, 'non-existent-file.pdf')
+        self.assertRaises(Exception, fulltext.get, 'non-existent-file.pdf')
```

Comment:
Why does this need to be the general exception?

Reply:
It does not.